schema.graphql

```diff
@@ -5,10 +5,15 @@ type Domain @entity {
   name: String
   labelHash: String
   parent: Domain
-  subdomains: [Domain!]
+  subdomains: [Domain!] @derivedFrom(field: "parent")
   owner: Account
   creator: Account
   transactionId: Bytes
+  lockedBy: Account
+  isLocked: Boolean
+  metaData: String
+  royalty: Int
+  events: [Domain!]!
```

Comment:
events should be of type `DomainEvent` not `Domain`

Reply:
This was actually what was breaking it after I tried adding the DomainEvent interface back in yesterday. completely removing this was what fixed it.